docs/develop/integrations/fastly.md

```diff
@@ -0,0 +1,186 @@
+    "MOMENTO_BACKEND": "<YOUR-MOMENTO-BACKEND-NAME>",
+    "MOMENTO_CACHE": "<YOUR-CACHE-NAME>",
+    "MOMENTO_TOKEN": "<YOUR-MOMENTO-AUTH-TOKEN>
+}
+```
+
+The name your assign to the variable `MOMENTO_BACKEND` can be anything you want, just make sure that your HTTP endpoint corresponds to the region where you created your Momento Cache. This is the HTTP we copied from the Momento Console.
+
+**Note**: for production environments, the Momento auth token should be saved in a [Fastly Secret Store](https://developer.fastly.com/reference/api/services/resources/secret-store/). However, this is a feature currently restricted to beta users, so this example saves the auth token in a [Config Store](https://developer.fastly.com/reference/api/services/resources/config-store/) along with the other values specified in the `secrets.json`.
+
+Next, you'll want to make sure the contents of your `secrets.json` file match the contents in the `fastly.toml` file. 
+
+Specifically, you'll want to make sure your HTTP endpoint and backend name is provided under the `[local_server]` heading. The local execution environment will use the information supplied under this heading to set up the appropriate backend and config store for your localhost server while you're developing.
+
+```
+[local_server]
+
+  [local_server.backends]
+
+    [local_server.backends.<YOUR-MOMENTO-BACKEND-NAME>]
+      url = "https://<YOUR-HTTP-ENDPOINT>"
+
+  [local_server.config_stores]
+
+    [local_server.config_stores.secrets]
+      file = "secrets.json"
+      format = "json"
+```
+
+Finally, you'll want to make sure all four variables in the `secrets.json` file are copied over to the items under the `[setup]` heading. Fastly will use the information supplied under this heading to create and initialize the appropriate backend and config store resources in the execution environment. 
+
+```
+[setup]
+
+  [setup.backends]
+
+    [setup.backends.<YOUR-MOMENTO-BACKEND-NAME>]
+      address = "<YOUR-HTTP-ENDPOINT>"
+      port = 443
+
+  [setup.config_stores]
+
+    [setup.config_stores.secrets]
+
+      [setup.config_stores.secrets.items]
+
+        [setup.config_stores.secrets.items.MOMENTO_BACKEND]
+          value = "<YOUR-MOMENTO-BACKEND-NAME>"
+
+        [setup.config_stores.secrets.items.MOMENTO_CACHE]
+          value = "<YOUR-CACHE-NAME>"
+
+        [setup.config_stores.secrets.items.MOMENTO_HTTP_ENDPOINT]
+          value = "<YOUR-HTTP-ENDPOINT>"
+
+        [setup.config_stores.secrets.items.MOMENTO_TOKEN]
+          value = "<YOUR-MOMENTO-AUTH-TOKEN>"
+```
+
+Next, start the development server and navigate to [localhost:7676](http://localhost:7676) to check that it's working.
+
+```bash
+fastly compute serve
+```
+
+The code in this example sets an item in the cache, gets it, deletes it, and returns an HTML response. 
+What you should see is a simple text response that matches the result seen in the [deployed example](https://solely-civil-crab.edgecompute.app/).
```

Comment:
I think we should remove the link to the deployed app since it is not permanent, and replace it with a screenshot.